### PR TITLE
Re-enable WorkerStatus report in Worker

### PR DESCRIPTION
**Pull request by Hazhzeng**
### Description

Previously, due to an issue where the latest request id is not sent back to the host, the /admin/host/ping endpoint will always timeout with 408 response. This results in over-scaling behavior in Python function apps. We disabled the WorkerStatus to fix the regression.

In this PR, we're going to re-enable the worker status and fix the request_id that it now should propagate correctly. Adding more unittests and endtoend tests to prevent the issue from happening again.

### Changes

1. Adding a new file to run endtoend tests for Http functions. This file is using the same testing infrastructure as unittest/test_http_functions.py but the scenarios are more focus on overall goodness of endtoend flow.
2. Re-enable the WorkerStatus request handler. Also add a test case in endtoend Http functions to ping the admin/host/ping?checkhealth=1 to ensure the WorkerStatus is responded back to the host.

### Todo

1. Ensure the req.request_id is passed into the handlers correctly 
2. Review unittest cases

---

### PR information

- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] If applicable, the PR references the bug/issue that it fixes in the description.
- [x] New Unit tests were added for the changes made and CI is passing.

### Quality of Code and Contribution Guidelines
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-functions-python-worker/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: tests/endtoend/test_http_functions.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        """
+        root_url = self.webhost._addr
+        health_check_url = f'{root_url}/admin/host/ping'
+        r = requests.post(health_check_url,
+                          params={'checkHealth': '1'},
+                          timeout=REQUEST_TIMEOUT_SEC)
+        self.assertTrue(r.ok)
+
+    @testutils.retryable_test(3, 5)
+    def test_worker_status_endpoint_should_return_ok_when_disabled(self):
```
Comment (Hazhzeng) [inline tests/endtoend/test_http_functions.py:85]: This is the test to ensure the admin/host/ping endpoint is not blocked

Comment (vrdmr) [reply tests/endtoend/test_http_functions.py:85]: Good one to test

**Review by vrdmr — APPROVED** [review]
LGTM.

File: azure_functions_worker/logging.py [hunk 2]
```diff
@@ -78,4 +79,6 @@ def enable_console_logging() -> None:
 def is_system_log_category(ctg: str) -> bool:
     # Category starts with 'azure_functions_worker' or
     # 'azure_functions_worker_errors' will be treated as system logs
-    return ctg.lower().startswith(SYSTEM_LOG_PREFIX)
+    ctg_lower = ctg.lower()
+    return ctg_lower.startswith(SYSTEM_LOG_PREFIX) or (
+        ctg_lower.startswith(SDK_LOG_PREFIX))
```
[the remark this answers is not in the document]
Comment (Hazhzeng) [reply azure_functions_worker/logging.py]: Stefanus may have more insights how this suppose to work. Hi @stefanushinardi, do you think we should send out WSGI and ASGI logs by default, or should we add a new feature flag to ask our customers to send it to us?

File: tests/endtoend/table_functions/table_in_binding/function.json [hunk 3]
```diff
@@ -1,28 +1,28 @@
...
+        "get"
+      ],
+      "name": "req"
+    },
+    {
+      "direction": "in",
+      "type": "table",
+      "name": "testEntity",
+      "partitionKey": "test",
+      "rowKey": "86a0d7b5-d130-41e5-8d34-24a6517770b0",
```
[the remark this answers is not in the document]
Comment (Hazhzeng) [reply tests/endtoend/table_functions/table_in_binding/function.json]: Seems like this is an old commit, the new one doesn't have it anymore.

**Pull request merged**